packages/@uppy/utils/src/truncateString.test.js

```diff
@@ -12,20 +12,15 @@ describe('truncateString', () => {
     expect(truncateString('abcdefghijkl', 7)).toEqual('ab...kl')
     expect(truncateString('abcdefghijkl', 6)).toEqual('ab...l')
     expect(truncateString('abcdefghijkl', 5)).toEqual('a...l')
-    expect(truncateString('abcdefghijkl', 4)).toEqual('a...')
-    expect(truncateString('abcdefghijkl', 3)).toEqual('abc')
-    expect(truncateString('abcdefghijkl', 2)).toEqual('ab')
-    expect(truncateString('abcdefghijkl', 1)).toEqual('a')
+    expect(truncateString('abcdefghijkl', 4)).toEqual('abc…')
```

Comment:
Will this new truncation have potential unforeseen changes in the UI somewhere?

Reply:
For reference, the only place `truncateString` is used in the code base is here: https://github.com/transloadit/uppy/blob/2f791c1747306352a20ab912b906394752a815bc/packages/@uppy/dashboard/src/components/FileItem/FileInfo/index.js#L20-L22
And `maxNameLength` is never under `30`: https://github.com/transloadit/uppy/blob/2f791c1747306352a20ab912b906394752a815bc/packages/@uppy/dashboard/src/components/FileItem/FileInfo/index.js#L9-L17